pkg/platform/runtime2/setup/implementations/alternative/artifact.go

```diff
@@ -45,3 +45,7 @@ func (as *ArtifactSetup) Move(tmpDir string) error {
 func (as *ArtifactSetup) Unarchiver() unarchiver.Unarchiver {
 	return unarchiver.NewTarGz()
 }
+
+func (as *ArtifactSetup) InstallerExtension() string {
```

Comment:
Let's remove this and have the Unarchiver instead return an interface that implements` unarchiver.Unarchiver` as well as `Ext() string`.